src/utils/exportUtils/PlainTextExport.ts

```diff
@@ -103,8 +103,9 @@ export default class PlainTextExporter extends Exporter {
                 }
             } else mediaText = ` (${this.mediaOmitText})`;
         }
-        if (this.isReply(mxEv)) return senderDisplayName + ": " + this.textForReplyEvent(mxEv.getContent()) + mediaText;
-        else return textForEvent(mxEv, this.room.client) + mediaText;
+        if (this.isReply(mxEv))
+            return `<${senderDisplayName}>` + ": " + this.textForReplyEvent(mxEv.getContent()) + mediaText;
+        else return textForEvent(mxEv, this.room.client, true, true) + mediaText;
```

Comment:
This appears to set `allowJSX` and `showHiddenEvents` to `true` where neither was true before? Is that deliberate? If so, why?